lib/bindings/python/src/dynamo/runtime/logging.py

```
@@ -213,7 +213,7 @@ def configure_sglang_logging(dyn_level: int):
         os.environ["SGLANG_LOGGING_CONFIG_PATH"] = f.name
 
 
-def configure_vllm_logging(dyn_level: int):
+def configure_vllm_logging(dyn_level: int) -> None:
```

Ruff (0.15.2)

[warning] Unused function argument: `dyn_level`

(ARG001)